homeassistant/components/glances/const.py

```diff
@@ -36,7 +36,9 @@
     "process_thread": ["processcount", "Thread", "Count", CPU_ICON],
     "process_sleeping": ["processcount", "Sleeping", "Count", CPU_ICON],
     "cpu_use_percent": ["cpu", "CPU used", PERCENTAGE, CPU_ICON],
-    "sensor_temp": ["sensors", "Temp", TEMP_CELSIUS, "mdi:thermometer"],
```

Comment:
Removing this sensor type looks like a breaking change. Please confirm.

Reply:
Hi, it's a "sensor type" in the sense of Glances API, which returns an array of "sensors". But they have nothing to do with the sensors in the Home Assistant sense.

However, the Entity IDs DO change. Now, I don't know if that one should be considered as breaking change or not...

Reply:
Yes, changing entity id is a breaking change. If possible we should avoid the breaking change. Otherwise we need to add a breaking change paragraph.

Reply:
Hi, well we could avoid Entity ID changes on the temperatures by reverting the param 2 to "Temp".